.eslintrc

```diff
@@ -1,8 +1,14 @@
 {
   "extends": [
     "airbnb",
-    "airbnb/hooks"
+    "airbnb/hooks",
+    "plugin:react/recommended",
```

Comment:
I'm not sure which value these added "extends" give.

Reply:
It seems to be really nothing.